### PR TITLE
#113 Validate each PrescriptionIndex prescriptions from the submission file

### PR DESCRIPTION
**Pull request by ofrancon**
Prescriptions file contain 1 set of data per PrescriptionIndex.
Validate each set independently

### CODE REVIEW

File: covid_xprize/validation/prescriptor_validation.py [hunk 1]
```diff
@@ -44,22 +44,28 @@ def validate_submission(start_date: str,
...
     if not all_errors:
-        # Columns are good, check we got prescriptions for each requested country / region
-        all_errors += _check_geos(ip_df, presc_df)
-        # Check the IP values
-        all_errors += _check_prescription_values(presc_df)
-    #     # Check the prediction dates are correct
-        all_errors += _check_days(start_date, end_date, presc_df)
+        # For each individual prescription in the prescriptions file
+        prescription_indexes = presc_df.PrescriptionIndex.unique()
+        for i in prescription_indexes:
```
Comment (ofrancon) [inline covid_xprize/validation/prescriptor_validation.py:61]: Validate each index independently

File: covid_xprize/validation/tests/test_prescriptor_validation.py [hunk 2]
```diff
@@ -68,3 +69,7 @@ def test_dates(self):
                            'Aruba: Expected prediction for date 2020-08-03 but got 2020-08-04',
                            'Aruba: Expected prediction for date 2020-08-04 but got None']
         self.assertEqual(expected_errors, errors)
+
+    def test_multi_prescription_index(self):
```
Comment (ofrancon) [inline covid_xprize/validation/tests/test_prescriptor_validation.py:73]: Add a test for submission files with multiple PrescriptionIndex.

File: covid_xprize/validation/prescriptor_validation.py [hunk 3]
```diff
@@ -44,22 +44,28 @@ def validate_submission(start_date: str,
     presc_df = pd.read_csv(submission_file,
                            parse_dates=['Date'],
                            encoding="ISO-8859-1",
```
Comment (EKMeyerson) [inline covid_xprize/validation/prescriptor_validation.py:46]: this is a nice pattern; i'll try to start using it

**Review by EKMeyerson — APPROVED** [review]
lgtm

**Pull request merged**